README.md

```diff
@@ -1,115 +1,306 @@
-# nri - Node Resource Interface
+## Node Resource Interface, Revisited
 
-[![PkgGoDev](https://pkg.go.dev/badge/github.com/containerd/nri)](https://pkg.go.dev/github.com/containerd/nri)
```

Comment:
let's leave this section at the top level and leave the widgets here..

Reply:
Moved back to README.md.